client/src/components/AdventModal.js

```diff
@@ -84,3 +35,26 @@ const Button = styled.button`
   border: none;
   cursor: pointer;
 `;
+
+const ModalStyle = styled.div`
+  background: radial-gradient(
+    circle,
+    rgba(210, 68, 68, 1) 0%,
+    rgba(0, 0, 0, 0.8) 70%
+  );
+  border: 1px solid var(--third);
+  border-radius: 0.5rem;
+  display: grid;
+  max-height: 100%;
+  max-width: 100%;
+  place-items: center;
+  padding: 0.5rem;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  text-align: center;
+  transform: translate(-50%, -50%);
+  box-shadow: 15px 0px 150px 12px rgba(255, 255, 255, 0.8),
```

Comment:
In line 45, you refer to a color var(--third). You could do the same in line 57: define a new var and use it here.